ui/Modal/ManagePeers.svelte

```diff
@@ -92,7 +92,7 @@
           dataCy="peer-input"
           hint="v"
           bind:value={newPeer}
-          placeholder="Paste a remote address here"
+          placeholder="Paste a peer address here..."
```

Comment:
```suggestion
          placeholder="Paste a peer address here…"
```

We use ellipses everywhere else.